setup/package.py

```diff
@@ -372,6 +358,21 @@ def start_test(self):
         self.test_odoo()
         logging.info('Finished testing rpm package')
 
+    def gen_rpm_repo(self, args, rpm_filepath):
```

Comment:
So this is just the bits removed from the new rpm_sign? It basically just split the old gen_rpm_repo in two?

Any reason why this is now a method on DockerRpm and rpm_sign is not?

Reply:
Because rpm_sign needs gpg stuffs not directly accessible in the Docker container... unless I do a big refactoring of the Docker class.

Reply:
OK.